pkg/services/ngalert/provisioning/notification_policies.go

```diff
@@ -70,6 +70,7 @@ func (nps *NotificationPolicyService) UpdatePolicyTree(ctx context.Context, orgI
 		return err
 	}
 
+	receivers[""] = struct{}{} // Allow empty receiver (inheriting from parent)
```

Comment:
this is good enough for a bugfix. Ideally, we should update the validation method.

Reply:
heh, didn't know if it belonged in the external lib or here. In any case, the test now validates it, so it can be changed safely